sbin/pfdhcplistener

```diff
@@ -179,7 +187,7 @@ sub dhcp_detector {
 
 sub process_pkt {
     my ( $user_data, $hdr, $pkt ) = @_;
-    listen_dhcp( $pkt, $user_data );
+    listen_dhcp( $pkt, $user_data ) if (($process eq '0') || ($process eq 'master'));
```

Comment:
Should also check if it's in cluster

so should be : 
if($process || !$cluster_enabled)